tests/checks/profiles.nf/.expected_profile_advanced.txt

```diff
@@ -1,5 +1,4 @@
-echo = true
-foo = 'bar'
+outputDir = 'results'
```

Comment:
Had to pick some valid config option here. Alternatively we could leave the invalid options and include the warnings for them in the expected stdout